src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costOfDay = 40;
+  const off7 = 50;
+  const off3 = 20;
+  let total = costOfDay * days;
+
+  if (days >= 7) {
+    total = total - off7;
+  } else if (days >= 3) {
+    total = total - off3;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    return total - off7;
  } 
  
  if (days >= 3) {
    return total - off3;
  }
```
As for me, it looks better. Btw, you can also add variables for 3 and 7, as we can change the required amount of days for the discount later.